app/forms/task_form.rb

```diff
@@ -0,0 +1,9 @@
+class TaskForm
+  include ActiveModel::Model
+
+  attr_accessor :title, :description
+
+  validates :title,
+    presence: true,
```

Comment:
Style/AlignParameters: Align the parameters of a method call if they span more than one line.